### PR TITLE
makes it so HeaderContent accepts an array of metaItems

### PR DESCRIPTION
**Pull request by rosscreighton**

### CODE REVIEW

File: imports/ui/components/AlbumPage/AlbumPage.jsx [hunk 1]
```diff
@@ -20,8 +20,10 @@ function AlbumPage({
         <HeaderContent
           header={album.name}
           subHeader={artist.name}
-          reviewCount={album.reviewCount}
-          averageRating={album.averageRating}
+          metaItems={[
```
Comment (sameckmeier) [inline imports/ui/components/AlbumPage/AlbumPage.jsx]: 👍 hell ya

File: imports/ui/components/HeaderContent/HeaderContent.jsx [hunk 2]
```diff
@@ -38,8 +35,10 @@ HeaderContent.propTypes = {
   imageUrl: PropTypes.string,
   header: PropTypes.string.isRequired,
   subHeader: PropTypes.string.isRequired,
-  reviewCount: PropTypes.number.isRequired,
-  averageRating: PropTypes.number.isRequired,
+  metaItems: PropTypes.arrayOf(PropTypes.shape({
+    label: PropTypes.string.isRequired,
+    value: PropTypes.oneOf([PropTypes.string, PropTypes.number]).isRequired,
```
Comment (sameckmeier) [inline imports/ui/components/HeaderContent/HeaderContent.jsx]: what are your thoughts about adding a className key?

File: imports/ui/stylesheets/components/HeaderContent.scss [hunk 3]
```diff
@@ -72,6 +72,7 @@
 
   :first-child {
```
Comment (sameckmeier) [inline imports/ui/stylesheets/components/HeaderContent.scss]: per previous comment, using a className feels more descriptive and flexible.

File: imports/ui/components/HeaderContent/HeaderContent.jsx (hunk 2 above)
Comment (rosscreighton) [inline imports/ui/components/HeaderContent/HeaderContent.jsx]: Do you think we need it yet? All of the meta items look the same at this point. I don't think we need to customize them with a class name.

File: imports/ui/stylesheets/components/HeaderContent.scss (hunk 3 above)
Comment (rosscreighton) [inline imports/ui/stylesheets/components/HeaderContent.scss]: Ohhhhhh

Comment (rosscreighton) [inline imports/ui/stylesheets/components/HeaderContent.scss]: Yeah I used className in a later commit. Youre right I was trying to be a little too clever here.

**Pull request merged**